docs/rules/cookbook/pandas-cookbook.md

```
@@ -511,33 +613,40 @@ d["fault_confirmed"] = confirm_fault(d["fault_raw"], min_rows=max(1, FAULT_CONFI
 
 | Param | Label | Unit | Default | Range |
 |-------|-------|------|--------:|-------|
-| `airflow_err` | OA fraction error | frac | 0.15 | 0.05–0.5 |
+| `eps_airflow` | Airflow error εF (GL36 default 30%) | frac | 0.15 | 0.05–1.0 |
+| `delta_t_min` | Minimum |OAT−RAT| ΔTmin (GL36 default 10°F) | °F | 5.0 | 0.0–30.0 |
```

markdownlint-cli2 (0.23.0)

[warning] Table column count
Expected: 5; Actual: 7; Too many cells, extra data will be missing

(MD056, table-column-count)